src/components/AudioPlayer/Player.js

```diff
@@ -0,0 +1,428 @@
+      }
+    }
+    this.scrubStart = event => {
+      this.scrubbing = true
+      if (event.type === 'mousedown') {
+        const up = e => {
+          this.scrubEnd(e)
+          document.removeEventListener('mousemove', this.scrub)
+          document.removeEventListener('mouseup', up)
+        }
+        document.addEventListener('mousemove', this.scrub)
+        document.addEventListener('mouseup', up)
+      }
+      this.scrub(event)
+    }
+    this.scrubEnd = event => {
+      this.scrub(event)
+      this.scrubbing = false
+    }
+    this.setInstanceState = state => {
+      this.setState(state)
+    }
+  }
+  toggle() {
+    const { audio } = this
+    if (audio) {
+      if (audio.paused || audio.ended) {
+        this.play()
+      } else {
+        this.pause()
+      }
+    }
+  }
+  play() {
+    const { audio } = this
+    audio && audio.play()
+  }
+  pause() {
+    const { audio } = this
+    audio && audio.pause()
+  }
+  setFormattedTimes() {
+    if (!this.audio || !this.audio.duration) {
+      return
+    }
+    this.formattedDuration = getFormattedTime(this.audio.duration)
+    if (!this.formattedCurrentTime) {
+      this.formattedCurrentTime = getFormattedTime(0)
+    }
+  }
+  componentDidMount() {
+    globalState.instances.push(this.setInstanceState)
+    if (!this.audio) {
+      return
+    }
+    this.audio.addEventListener('play', this.onPlay)
+    this.audio.addEventListener('pause', this.onPause)
+    this.audio.addEventListener('loadstart', this.onLoadStart)
+    this.audio.addEventListener('canplay', this.onCanPlay)
+    this.audio.addEventListener('canplaythrough', this.onCanPlay)
+    this.audio.addEventListener('loadedmetadata', this.onLoadedMetaData)
+
+    this.setFormattedTimes()
+
+    if (this.audio && !this.audio.paused) {
+      this.onPlay()
+    }
+  }
+  componentDidUpdate() {
+    this.setFormattedTimes()
+  }
+  componentWillUnmount() {
+    globalState.instances = globalState.instances.filter(
+      setter => setter !== this.setInstanceState
+    )
+    if (globalState.playingRef === this.audio) {
+      globalState.playingRef = undefined
+    }
+
+    this.audio.removeEventListener('play', this.onPlay)
+    this.audio.removeEventListener('pause', this.onPause)
+    this.audio.removeEventListener('loadstart', this.onLoadStart)
+    this.audio.removeEventListener('progress', this.onProgress)
+    this.audio.removeEventListener('canplay', this.onCanPlay)
+    this.audio.removeEventListener('canplaythrough', this.onCanPlay)
+    this.audio.removeEventListener('loadedmetadata', this.onLoadedMetaData)
+  }
+  render() {
+    const { src, size, attributes = {} } = this.props
+    const { playing, progress, muted, loading, buffered } = this.state
+    const isVideo = src.mp4 || src.hls
+
+    return (
+      <div {...merge(styles.wrapper, breakoutStyles[size])}>
+        {!isVideo && <audio
+          {...styles.audio}
+          {...attributes}
+          style={this.props.style}
```

Comment:
drop, relict from `VideoPlayer` that IMO has no audio use case (is for cover videos)